draft-ietf-quic-recovery.md

```diff
@@ -725,9 +725,9 @@ sent, and then the RTO timer is set.
 
 #### Early Retransmit Alarm
 
-Early retransmit {{?RFC5827}} is implemented with a 1/4 RTT timer. It is
-part of QUIC's time based loss detection, but is always enabled, even when
-only packet reordering loss detection is enabled.
+Early retransmit {{?RFC5827}} is only used when using Fast Retransmit in packet
+number mode. It is implemented with a 1/4 RTT timer. It is part of QUIC's time
```

Comment:
this seems a bit confusing, i think we could say something like

``We expect to receive feedback about the sent packet within 1 RTT of the time_since_sent. Early retransmit is implemented by setting a (1 + 1/4) RTT timer since the time the packet is sent. ```